src/i18n/en.json

```diff
@@ -860,7 +861,8 @@
       "funded": "{{funded, percent}} funded",
       "investors": "### {{investors}}\ninvestors",
       "days-to-go": "### {{days}}\ndays to go",
-      "view-details-btn": "View details"
+      "view-details-btn": "View details",
+      "days-after-the-end": "### {{days}}\ndays after closed"
```

Comment:
also pls remove this `###` part here and in two cases above. Localization files are the border territory and being shared between programmers and translators. Translators should not be responsible for markup that related to business logic, like the current one. We should extract that h3 tag out of translations and leave only "days after", "investors" and "days to go" part, the digit should be inserted as is in the new h3 tag, and right below the digit should be inserted translation of the label.

p.s.: looks like it merged by mistake, sorry for that